src/test/e2e/api/admin/projects.e2e.test.ts

```diff
@@ -0,0 +1,31 @@
+import dbInit, { ITestDb } from '../../helpers/database-init';
+import { IUnleashTest, setupApp } from '../../helpers/test-helper';
+import getLogger from '../../../fixtures/no-logger';
+let db: ITestDb;
+let app: IUnleashTest;
+
+beforeAll(async () => {
+    db = await dbInit('projects_api_serial', getLogger);
+    app = await setupApp(db.stores);
```

Comment:
It would be nice to test the schema within the test.

```
 app = await setupAppWithCustomConfig(db.stores, {
      experimental: {
          flags: {
              strictSchemaValidation: true,
          },
      },
  });
```

Reply:
Ah, nice. I will take a look.